packages/web/public/static/locales/de/my-posts.json

```diff
@@ -3,5 +3,6 @@
   "publishedTab": "Veröffentlicht",
   "draftsTab": "Entwürfe",
   "publishedEmptyState": "Du hast noch keine Beiträge veröffentlicht. Du kannst entweder einen Entwurf veröffentlichen oder <1>einen neuen Post</1>.",
-  "draftEmptyState": "Du hast keine Entwürfe."
+  "draftEmptyState": "Du hast keine Entwürfe.",
+  "errorLoadingPostsMessage": "There was an error retrieving your posts."
```

Comment:
Hi @LindasLinguas , I just added this translation that I found we weren't yet translating in the UI. I'm not sure if you'll be able to get to it in time as we are going to be finally deploying everything to production in the next couple of hours, but I thought I'd include it and tag you just in case!